tests/plugins/clipboard/manual/pastecommands.md

```diff
@@ -3,7 +3,7 @@
 @bender-ckeditor-plugins: clipboard, pastetext, pastefromword, wysiwygarea, toolbar, undo, elementspath, clipboard, floatingspace, sourcearea
 
 
-Have fun with clipboard plugins!
+Have fun with clipboard plugins! Use buttons in editor!
```

Comment:
The change is not precise enough, it should say at lest "clipboard buttons" as there are many other buttons. In that case first sequence is no longer needed.